src/_includes/feature_lists/cloud.njk

```diff
@@ -96,56 +119,16 @@
             label: "Certified Nodes",
             values: [null, 'check', 'check'],
             info: "<p>FlowFuse Certified Nodes a selection of certfied Node-RED nodes, enhancing efficiency and security.</p>"
-        }, {
-            id: "multi-user-dashboard",
-            label: "Personalised Multi-user Dashboards",
-            values: [null, 'check', 'check'],
-            info: "<p>Personalised Multi-user Dashboard allow you to build applications that provide unique data to each user <a href=\"https://flowfuse.com/blog/2024/01/dashboard-2-multi-user\" target=\"blank\">Learn more</a></p>"
         }, {
             id: "instance-monitoring",
             label: "Instance Monitoring",
             values: [null, 'check', 'check'],
             info: "<p>You can enable alerts to be sent to you via email. <a href=\"https://flowfuse.com/docs/user/instance-settings/#alerts\" target=\"blank\">Learn more</a></p>"
-        }, {
-            id: "device-groups",
-            label: "Device Group Management",
-            values: [null, null, 'check'],
-            info: "<p>Device Groups enables users to logically group their devices assigned to an application. Additionally, Device Groups can be integrated into the DevOps Pipeline.</p>"
         }, {
             id: "sso",
             label: "Single Sign-On (SSO)",
             values: [null, null, 'check'],
             info: "<p>FlowFuse can be configured to run with your own Single Sign-On (SSO) provider. Allow users to access FlowFuse with a single set of login credentials, improving security, user experience, and save them time.</p>"
-        }, {
-            id: "ha",
-            label: "High Availibility",
-            values: [null, null, 'check'],
-            info: "<p>Leverage horizontal scaling for reliable and scalible processing of your data through Node-RED.</p>"
-        }]
-    }, {
-        label: "Support & SLA",
```

Comment:
@zackwasli WDYT about removing support from the pricing page?

Reply:
Entirely? The plan was to remove it here and add it to the hero section, and I've done just that. How crucial is this aspect for potential customers when they evaluate which plan best suits their needs? If you say it isn't, I think we can remove it.

Reply:
If we're talking about this section, I think it displays useful info around support (specifically SLAs). What would be the reasoning to remove it?

<img width="955" alt="Screenshot 2024-04-26 at 10 07 08 AM" src="https://github.com/FlowFuse/website/assets/136500602/5ea79380-fde5-4f48-9c82-e2de6d08553a">

Reply:
The plan was to remove it here and to add only the SLA info in the hero section, and I've done just that.